DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/src/post_process.hpp

```diff
@@ -0,0 +1,259 @@
+#ifndef _POST_PROCESS_HPP_
+#define _POST_PROCESS_HPP_
+
+#include "constexpr_math.hpp"
+#include "tuple.hpp"
+#include "unrolled_loop.hpp"
+// #include "orthogonalizer.hpp"
+
+namespace fpga_svd {
+
+template <typename T, bool is_complex, int A_rows, int A_cols, int pipe_size,
+          typename AIn,   // Original SVD input A,        size A_rows x A_cols
+          typename RIn,   // R matrix from QR iteration,  size 1D - [A_cols]
+          typename VIn,   // Q accumilated V input,       size A_cols x A_cols
+          typename UOut,  // U output                     size A_rows x A_rows
+          typename SOut,  // S output                     size A_rows x A_cols
+          typename VOut>  // V output                     size A_cols x A_cols
```

Comment:
Even if the names of these pipes don't match the ones in the diagram, the comment should at least match 😊

```suggestion
          typename AIn,   // InputMatrixPipe2,        size A_rows x A_cols
          typename RIn,   // R matrix from QR iteration,  size 1D - [A_cols]
          typename VIn,   // Q accumilated V input,       size A_cols x A_cols
          typename UOut,  // UTempMatrixPipe                     size A_rows x A_rows
          typename SOut,  // SMatrixPipe                     size A_rows x A_cols
          typename VOut>  // VMatrixPipe                    size A_cols x A_cols
```
Also double-check the dimensions 😊

Reply:
I think some of these pipe names are not correct; e.g. `RIn` ends up mapped to `EigenValuesPipe` and `VIn` gets mapped to `EigenVectorsPipe`.